src/pages/eunjeonglee/Login/Login.js

```diff
@@ -1,9 +1,75 @@
 import React from 'react';
+import './Login.scss';
+
+class Login extends React.Component {
+  constructor() {
+    super();
+    this.state = {
+      id: '',
+      password: '',
+      logIn: false,
+    };
+  }
+  handleInput = event => {
+    const { name, value } = event.target;
+    this.setState({
+      [name]: value,
+    });
+  };
+
+  isValid = event => {
+    const { id, password } = this.state;
+    const isValidLogIn = id.includes('@') && password.length >= 5;
+    this.setState({
+      logIn: isValidLogIn,
+    });
+  };
+
+  goToList = () => {
+    if (this.state.logIn === true) {
+      this.props.history.push('/list-eunjeong');
+    }
+  };
 
-class Detail extends React.Component {
   render() {
-    return 'Detail';
+    const { handleInput, isValid, goToList, state } = this;
```

Comment:
state 안의 구조분해 할당은 따로 해주세요!
```suggestion
    const { handleInput, isValid, goToList } = this;
    const { logIn } = this.state;
```